packages/empty-state/package.json

```diff
@@ -0,0 +1,35 @@
+
+  {
+    "name": "@leafygreen-ui/empty-state",
+    "version": "0.9.0",
+    "description": "leafyGreen UI Kit Empty State",
+    "main": "./dist/index.js",
+    "module": "./dist/esm/index.js",
+    "types": "./dist/index.d.ts",
+    "typesVersions": {
+      "<3.9": { "*": ["ts3.4/*"] }
+    },
+    "scripts": {
+      "build": "../../node_modules/.bin/rollup --config ../../rollup.config.js"
```

Comment:
is it worth checking to see bundle size with and without the example-graphics -- i'm sure it's not too bad, but if the difference is not negligible maybe we can store these elsewhere?

Reply:
Yeah I don't expect it to be a problem - the largest graphic file is 7kb.